packages/sns/test/fakes/FakeConsumer.ts

```diff
@@ -0,0 +1,45 @@
+import type { Either } from '@lokalise/node-core'
+import type { BaseMessageType } from '@message-queue-toolkit/core'
+import { MessageHandlerConfigBuilder } from '@message-queue-toolkit/core'
+import type { SNSSQSConsumerDependencies } from '@message-queue-toolkit/sns'
+import type { ZodSchema } from 'zod'
+
+import { AbstractSnsSqsConsumer } from '../../lib/sns/AbstractSnsSqsConsumer'
+
+export class FakeConsumer<T extends BaseMessageType> extends AbstractSnsSqsConsumer<T, unknown> {
```

Comment:
Could make sense to expose this class to provide an easy way to replace real consumers on tests?

Reply:
will do!